src/controller/artifact/controller.go

```diff
@@ -19,6 +19,7 @@ import (
 	"context"
 	stderrors "errors"
 	"fmt"
+	"github.com/goharbor/harbor/src/controller/artifact/processor/wasm"
```

Comment:
please revise the import, it has to move this line into github.com/goharbor/harbor group

Reply:
new PR will fix this